src/models/applicationState.ts

```diff
@@ -62,6 +62,17 @@ export interface ITag {
     color: string;
 }
 
+/**
+ * @enum LOCAL - Local storage type
+ * @enum CLOUD - Cloud storage type
+ * @enum OTHER - Any other storage type
+ */
+export enum StorageType {
```

Comment:
Let's keep the enum keys as PascalCase vs. all uppercase to be consistent with other app enums.